content/en/docs/concepts/services-networking/service.md

```diff
@@ -142,9 +142,9 @@ for convenience, the `targetPort` is set to the same value as the `port`
 field.
 {{< /note >}}
 
-### Port definitions {#field-spec-ports}
+### Defining ports {#field-spec-ports}
```

Comment:
In a concept page, the subheadings are usually also concepts. I'd stick with “Port definitions”.

Reply:
I just find it better aligning with **Defining a service** subheading bit on the top

Reply:
I'd be worried that some people then think they need to `kubectl create port http` or something.

Reply:
How about “Services and ports” as the heading?

Reply:
ok I will revert the original subheading

Reply:
Hmm **Services and ports**  I think could mean more than what we are talking about in this section just present how service can select ports by names